custom/icds_reports/utils/__init__.py

```diff
@@ -832,13 +832,19 @@ def track_time(func):
     from custom.icds_reports.models import AggregateSQLProfile
 
     def get_async_indicator_time():
-        return AsyncIndicator.objects.exclude(date_queued__isnull=True)\
-            .aggregate(Max('date_created'))['date_created__max'] or datetime.now()
+        try:
+            return AsyncIndicator.objects.exclude(date_queued__isnull=True)\
+                .aggregate(Max('date_created'))['date_created__max'] or datetime.now()
+        except:
+            pass
```

Comment:
might be better to explicitly return None and catch a `django.db.Error` instead of all exceptions